composer.json

```
@@ -0,0 +1,44 @@
+{
+    "name": "bedrock/rate-limit-bundle",
+    "type": "symfony-bundle",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Bedrock",
+            "email": "opensource@bedrockstreaming.com",
+            "homepage": "https://tech.bedrockstreaming.com/"
+        }
+    ],
+    "config": {
+        "bin-dir": "bin",
+        "vendor-dir": "vendor",
+        "sort-packages": true
+    },
+    "require": {
+        "php": "7.4.*",
+        "ext-json": "*",
+        "doctrine/annotations": "^1.10.0",
+        "symfony/dependency-injection": "4.4.*",
+        "symfony/event-dispatcher": "4.4.*",
+        "symfony/http-foundation": "4.4.*",
+        "symfony/http-kernel": "4.4.*",
+        "symfony/config": "4.4.*"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.4",
```

```suggestion
        "phpunit/phpunit": "9.4.*",
```

to make sure the dev env will also work in a few months (and that we won't be able to use features from `9.5+` without noticing it)